ush/global_det/time_series.py

```diff
@@ -67,7 +67,7 @@ def plot_time_series(df: pd.DataFrame, logger: logging.Logger,
                      date_hours: list = [0,6,12,18], verif_type: str = 'pres', 
                      save_dir: str = '.', fix_dir: str = '.',
                      requested_var: str = 'HGT', 
-                     line_type: str = 'SL1L2', dpi: int = 300, 
+                     line_type: str = 'SL1L2', dpi: int = 150, 
                      confidence_intervals: bool = False,
```

Comment:
I merged, but I wanted to double check before I go any further.  I think that @YaliMao-NOAA had changed this to 300 dpi in order to reduce the length of her plot job.  Since this is a shared job, @DeannaSpindler-NOAA @AliciaBentley-NOAA , can this stay at 300 dpi?  Or Yali - is it OK if this goes to 150?  Since this is common code.

Reply:
This is a decision for the EVS managers.  It only affects the plot size - the smaller plots are the dpi=150.  Either way should run fine.